Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -494,6 +494,7 @@ def setScores(self, scores, labels, values, colors, rownames=None):
         self.__setup()
 
     def setRowNames(self, names):
+        ROW_NAMES_WIDTH = 200
```

Comment:
So it's better to have code removed further away from where it's used, in this case in one single, particular place?

The ridicule is rhetorical.